### ISSUE TITLE
Withdraw / Transfer methods missing after TS upgrade

### ISSUE THREAD

**Issue by kingkong404** [ccxt/ccxt#17167]
### Operating System

OSX

### Programming Languages

JS

### CCXT Version

3.0.4

### Description

<img width="260" alt="image" src="https://user-images.githubusercontent.com/4363358/224981371-64ec0deb-7710-4542-b7df-ac997a1c7427.png">

`Property 'withdraw' does not exist on type 'Exchange'.ts(2339)`

`Property 'transfer' does not exist on type 'Exchange'.ts(2339)`

### Code

```
  

```

**Comment by carlosmiei** [comment]
Hello @kingkong404 thanks for reporting it and sorry for the inconvenience, we will take a look asap. As a quick cast can you add the `as any` cast so you don't get blocked by this?

**Comment by kingkong404** [comment]
Hey @carlosmiei I've just realised a patch i did to fix other TS issues has now caused this issue. For all my exchange instances I was forcing them as an Exchange to get around errors when passing in the bybit object into a function expecting an exchange. 

```export const bybit = new ccxtpro.bybit({
  apiKey: process.env.BYBIT_API,
  secret: process.env.BYBIT_SECRET,
  enableRateLimit: true,
}) as Exchange```

**Comment by kingkong404** [comment]
Thanks @carlosmiei yes that also now seems to be fixed will close this now and cover new issues in another issue.
